### DISCUSSION TITLE
BirdNETPi.com

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-Pi#183] [Announcements]
Hello, everyone!

I've finally started putting together a website for this thing! Check it out and let me know if anyone has suggestions or wants to take over.

https://birdnetpi.com

Thanks to @svardsten53 for contributing the cover photo and @CaiusX for the awesome visuals that make a website worth having!

**Comment by nilspupils** [comment]
Just now had a look and i like it a lot! I did click on it before but got a security alert from our system (at work) so couldn't open the page. Just checked again and now it works....
Best, 
Nils
